frontends/web/src/components/dialog/dialog.tsx

```diff
@@ -45,16 +46,23 @@ class Dialog extends Component<Props, State> {
   public state: State = {
     active: false,
     currentTab: 0,
+    isMobile: false,
   };
 
   public componentDidMount() {
     setTimeout(this.activate, 10);
+    this.checkIfMobile();
+    window.addEventListener('resize', this.checkIfMobile);
   }
 
   public componentWillUnmount() {
     document.removeEventListener('keydown', this.handleKeyDown);
   }
 
+  private checkIfMobile = () => {
+    this.setState({ isMobile: window.innerWidth <= 768 });
+  };
+
```

Comment:
We could use the newly created (also in this PR) `useMatchMedia` here once this component gets converted into a functional one.